source/backend/api/appsettings.json

```diff
@@ -4,20 +4,41 @@
     "Port": "8080",
     "LivePath": "/health/live",
     "ReadyPath": "/health/ready",
+    "PimsDbCollation": {
+      "Enabled": true,
+      "Duration": 1
+    },
+    "SqlServer": {
+      "Enabled": true,
+      "Duration": 1
+    },
+    "ApiMetrics": {
+      "Enabled": true,
+      "Duration": 1
+    },
     "PmbcExternalApi": {
       "Url": "https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW/wfs?service=WFS&REQUEST=GetFeature&VERSION=1.3.0&outputFormat=application/json&typeNames=pub:WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW&srsName=EPSG:4326&count=0",
-      "StatusCode": 200
+      "StatusCode": 200,
+      "Enabled": true,
+      "Duration": 1
     },
     "Geoserver": {
       "Url": "/ows?service=WFS&REQUEST=GetFeature&VERSION=1.3.0&outputFormat=application/json&typeNames=PIMS_PROPERTY_LOCATION_VW&srsName=EPSG:4326&count=0",
-      "StatusCode": 200
+      "StatusCode": 200,
+      "Enabled": true,
+      "Duration": 1
     },
     "Ltsa": {
-      "Pid": "6209521"
+      "Enabled": true,
+      "Duration": 1
```

Comment:
I see a discrepancy here in regards to the duration/period. In other configuration files I see the override values as "Period" vs here they are set originally as "Duration". I think you need to pick one...
![image](https://github.com/bcgov/PSP/assets/24322224/96b14c7f-4ae6-4978-9cc6-d06e502c46f9)